manifests/2.0.0/opensearch-2.0.0.yml

```diff
@@ -7,6 +7,7 @@ ci:
 build:
   name: OpenSearch
   version: 2.0.0
+  qualifier: alpha1
```

Comment:
What will be the qualifier for GA or prod artifact? Will this field be empty?

Reply:
Yes, we'll just remove this.

Reply:
In that case this gradle command having a null or empty value for qualifier will not be an issue?
https://github.com/opensearch-project/opensearch-build/pull/1731/files#diff-004d8135f0e1267ff669721dc64b44f84729dc6fe871c6c2ef02eb9d1e04959dR75

Reply:
It doesn't seem like it would be a problem, similarly we don't always pass a value for `-Dbuild.snapshot=`.

Reply:
Okay. Also do we need to have a check to allow only certain values as a qualifier?

Reply:
^Maybe not. This qualifier check logic is already handled [downstream](https://github.com/opensearch-project/OpenSearch/blob/044f53630a08b0021097e422c9f31f15fb98219d/buildSrc/build.gradle#L301-L302) in the component build, so adding it here seems redundant and creates two sources of truth. So good to skip imo.

Reply:
Looks like it only supports `(alpha|beta|rc)`

Reply:
I'm not sure if plugins build script inherits the qualifier check logic as well (?) If not, it might make sense to add here.

Reply:
Build will fail downstream, but probably makes sense to have a check here to fail on PRs to the manifest (fail earlier). Will add in a future PR.